pep/app/pods/components/radio-input/component.ts

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+
+interface RadioInputArgs {
+    onChange?: (newValue: boolean, event: Event) => void;
+    value: any;
+    inline?: boolean;
+    checked?: boolean;
+    disabled?: boolean;
+    label?: string;
+    preventDefault?: boolean;
+    stopPropagation?: boolean;
+}
+
+export default class RadioInput extends Component<RadioInputArgs> {
+    inputId: string = `radio-input-${guidFor(this)}`;
+
+    /**
+     * Invoke the radio's `onChange` method if it was passed in
+     * as an argument
+     *
+     */
+    @action
+    onChange(event: Event) {
+        if (this.args.preventDefault) {
```

Comment:
Thoughts about setting defaults here - to whatever the most common use case is

Reply:
do you mean making getters for these? like:
```ts
get preventDefault() {
    return this.args. preventDefault ?? false;
}
```
the default/most common case for both is `false`, thats why i didnt create getters, but i can if we wanna be more explicit.

for `onChange`, i can change that to just
```ts
this.args.onChange?.(this.args.value, event);
```